language/stdlib/modules/Roles.move

```diff
@@ -282,65 +174,75 @@ module Roles {
         // A role cannot have previously been assigned to `new_account`.
         assert(!exists<RoleId>(Signer::address_of(new_account)), 1);
         move_to(new_account, RoleId { role_id: UNHOSTED_ROLE_ID() });
-        move_to(new_account, Privilege<UnhostedRole>{ witness: UnhostedRole{}, is_extracted: false })
     }
 
-    ///////////////////////////////////////////////////////////////////////////
-    // Capability Extraction from Privileges, and Restoration to Privileges
-    ///////////////////////////////////////////////////////////////////////////
+    ///  ## privilege-checking functions for roles ##
+    ///
+    ///>TODO (dd) remove spreadsheet link unless we're going to publish it.
```

Comment:
Do this before landing perhaps? I do think we'll eventually have a CSV of the spreadsheet in the public Roles LIP